src/peer/negotiator.js

```diff
@@ -58,7 +59,7 @@ class Negotiator extends EventEmitter {
   async startConnection(options = {}) {
     this._pc = this._createPeerConnection(options.pcConfig);
     this._setupPCListeners();
-    this._originator = options.originator;
+    this.originator = options.originator;
```

Comment:
this can be encapsulated like

- `this._originator`
-  Negotiator.getOriginator() to get variable

I don't have strong opinion around this because the SDK isn't so large and can be maintained if I use public variable.